common/src/main/java/com/wynntils/screens/OverlayManagementScreen.java

```diff
@@ -0,0 +1,651 @@
+                    .collect(Collectors.toSet());
+
+            for (Overlay overlay : overlays) {
+                CustomColor color = overlay == selectedOverlay ? CommonColors.LIGHT_BLUE : CommonColors.GREEN;
+                RenderUtils.drawRectBorders(
+                        poseStack,
+                        color,
+                        overlay.getRenderX(),
+                        overlay.getRenderY(),
+                        overlay.getRenderX() + overlay.getWidth(),
+                        overlay.getRenderY() + overlay.getHeight(),
+                        1,
+                        1.8f);
+                RenderUtils.drawRect(
+                        poseStack,
+                        color.withAlpha(30),
+                        overlay.getRenderX(),
+                        overlay.getRenderY(),
+                        0,
+                        overlay.getWidth(),
+                        overlay.getHeight());
+
+                int xOffset =
+                        switch (overlay.getRenderHorizontalAlignment()) {
+                            case Left -> 3;
+                            case Center -> 0;
+                            case Right -> -3;
+                        };
+
+                int yOffset =
+                        switch (overlay.getRenderVerticalAlignment()) {
+                            case Top -> 5;
+                            case Middle -> 0;
+                            case Bottom -> -5;
+                        };
+
+                FontRenderer.getInstance()
+                        .renderTextWithAlignment(
+                                poseStack,
+                                overlay.getRenderX() + xOffset,
+                                overlay.getRenderY() + yOffset,
+                                new TextRenderTask(
+                                        overlay.getTranslatedName(),
+                                        new TextRenderSetting(
+                                                overlay.getWidth(),
+                                                color,
+                                                FontRenderer.TextAlignment.fromHorizontalAlignment(
+                                                        overlay.getRenderHorizontalAlignment()),
+                                                FontRenderer.TextShadow.OUTLINE)),
+                                overlay.getRenderedWidth() + xOffset,
+                                overlay.getRenderedHeight() + yOffset,
+                                overlay.getRenderHorizontalAlignment(),
+                                overlay.getRenderVerticalAlignment());
+            }
+
+            if (isMouseHoveringOverlay(selectedOverlay, mouseX, mouseY) && selectionMode == SelectionMode.None) {
+                RenderUtils.drawTooltipAt(
+                        poseStack,
+                        mouseX
+                                - RenderUtils.getToolTipWidth(
```

Comment:
Same here, extract tooltipWidth to a local variable to help formatting (and readability).